drupal/recipes/wunder_next_setup/config/rest.resource.webform_rest_submit.yml

```diff
@@ -15,4 +15,4 @@ configuration:
   formats:
     - json
   authentication:
-    - cookie
+    - oauth2
```

Comment:
This needs to be set to oauth2 so we can check the identity of the user doing the call to submit a webform